packages/itwinui-css/src/utils/icon.scss

```diff
@@ -11,16 +11,17 @@ $icon-sizes: (
 );
 
 $default-size: #{map.get($icon-sizes, m)};
-$default-fill: var(--iui-color-icon-muted);
 
 .iui-svg-icon {
+  --_iui-svg-icon-fill-default: var(--iui-color-icon-muted);
```

Comment:
I should have clarified: I'm not suggesting changing the global variables at the component level; I'm asking why it's not enough to change these variables globally. What is the use case for `--_iui-svg-icon-fill-default` needing a different default value from what is set globally?

Reply:
> That's because iTwinUI v3's default icon color is the muted one. However, in iTwinUI v5, I see that the default icon color is `--kiwi-color-icon-neutral-base`. Globally:

This is not completely true. In Kiwi, we do not even set a default Icon color; it just uses `currentColor`. I would say it's fine to kepe using secondary instead of base. So we don't need this variable right now.